src/api/Contracts/Tracer.php

```diff
@@ -20,6 +20,7 @@ interface Tracer
      * Tracer needs to handle any operation even if its in limited mode,
      * however users can opt not to use tracer when its in limited mode.
      *
+     * @deprecated Has no active functionality anymore and will always return true
```

Comment:
What alternative method we can provide to let customers using the open tracing inspired api to know when they should create or not a span? At the moment, if I am reading the code correctly, the extension-level limited check only applies to the `DDTrace\trace_*` functions.

In case this is correct, we should remove this deprecation notice.

Reply:
Don't know, at least I thought the ticket was clear in that way that we don't want to retain that functionality.

Reply:
To clarify from the card: we call closures passed to the [sandbox API](https://docs.datadoghq.com/tracing/setup_overview/custom_instrumentation/php/?tab=tracingfunctioncalls) "tracing closures". The integrations instrument function calls via tracing closures in the sandbox API. These are where the redundant limited checks were occurring.

Reply:
So what exactly shall I do here?

Reply:
We should remove the `@deprecated` tag since we're keeping the limited check in `Tracer::limited()`.